dockerfiles/site/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Build James
+#
+# VERSION	1.0
+
+FROM java:openjdk-6-jdk
+
+ENV GIT_VERSION 1:1.7.10.4-1+wheezy1
```

Comment:
we should not fix the git version, we actually don't care and it will break in the future